test/run-drun/map-upgrades.drun

```diff
@@ -3,16 +3,17 @@ ingress $ID go "DIDL\x00\x00"
 upgrade $ID map-upgrades/map0.mo ""
 ingress $ID upgradeNodes "DIDL\x00\x00"
 ingress $ID go "DIDL\x00\x00"
-upgrade $ID map-upgrades/map0.mo ""
-ingress $ID upgradeNodes "DIDL\x00\x00"
-ingress $ID go "DIDL\x00\x00"
-upgrade $ID map-upgrades/map0.mo ""
-ingress $ID upgradeNodes "DIDL\x00\x00"
-ingress $ID go "DIDL\x00\x00"
+# Disabled because of IC trap (too many open files): 
+# 'Failed to make a checkpoint @293: Persistence(FileSystemError { path: "/tmp/ic.XXX/state/checkpoints/0000000000000125/canister_states/00000000000000070101/stable_memory.bin", context: "Failed to open file", internal_error: "Too many open files (os error 24)" })', rs/state_manager/src/lib.rs:2330:25
```

Comment:
Oh dear.

Reply:
Oh no, embarrassing. Sorry that I missed this issue in the diff...

Reply:
Unfortunately. I will retest to see whether the error still (occasionally) appears.